store/src/sql/access/stream.rs

```diff
@@ -0,0 +1,147 @@
+use anyhow::anyhow;
+use cid::Cid;
+
+use crate::{
+    sql::entities::{
+        EventHeader, EventHeaderRow, EventType, IncompleteStream, StreamCid, StreamCommitRow,
+        StreamRow,
+    },
+    Error, Result, SqlitePool, SqliteTransaction,
+};
+
+/// Access to the stream and related tables. Generally querying events as a stream.
+pub struct CeramicOneStream {}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// Represents a stream event in a way that allows ordering it in the stream. It is metadata and not the event payload itself.
+pub struct StreamCommit {
```

Comment:
open to a better name

Reply:
yeah, I 100% was just trying to avoid `event` and used some js-ceramic (and git) language. I like `StreamEventMetadata` or similar - I think I even named it something like that at first but changed it because I didn't want to see Event again 🙈.